packages/next/build/swc/Cargo.toml

```diff
@@ -15,11 +15,11 @@ path-clean = "0.1"
 regex = "1.5"
 serde = "1"
 serde_json = "1"
-swc = "0.30"
+swc = "0.33"
 swc_atoms = "0.2.6"
 swc_common = { version = "0.11.1", features = ["tty-emitter", "sourcemap"] }
 swc_node_base = "0.2.0"
-swc_ecmascript = { version = "0.48.0", features = ["codegen", "optimization", "parser", "react", "transforms", "typescript", "utils", "visit"] }
+swc_ecmascript = { version = "0.50.0", features = ["codegen", "optimization", "parser", "react", "transforms", "typescript", "utils", "visit"] }
```

Comment:
I think `swc_ecmascript` should reexport `swc_ecma_preset_env`.
I'll do it soon.